whitelist.yml

```diff
@@ -2,18 +2,31 @@
 ap-southeast:
   - 52.62.30.119
   - 52.62.3.168
+  - 54.66.51.34
+  - 54.79.207.8
+  - 52.65.195.15
+  - 52.62.240.64
+  - 54.252.43.78
+  - 54.79.124.27
 eu-central:
-  - 52.29.64.156
-  - 18.196.195.93
```

Comment:
Do we want to remove these two ip addresses a bit later so that we can roll back easily?
52.29.64.156
18.196.195.93

Reply:
these are not being used